src/main/java/com/kmush12/CurrencyConverter/Json/JsonProcessor.java

```diff
@@ -0,0 +1,31 @@
+package com.kmush12.CurrencyConverter.Json;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.kmush12.CurrencyConverter.Calculator.AmountCalculator;
+import com.kmush12.CurrencyConverter.Exchange.ExchangeRate;
+import com.kmush12.CurrencyConverter.Exchange.ExchangeRequest;
+import com.kmush12.CurrencyConverter.Exchange.ExchangeResult;
+import org.springframework.stereotype.Component;
+
+@Component
+public class JsonProcessor {
+
+    private final AmountCalculator calculator;
+
+    public JsonProcessor(AmountCalculator calculator) {
+        this.calculator = calculator;
+    }
+
+    public JsonNode ListOfCurrencies(JsonNode node){
+            return node.get("symbols");
+        }
+
+        public ExchangeResult getExchangeResult(ExchangeRequest exchangeRequest, ExchangeRate exchangeRate){
```

Comment:
To nie jest procesowanie JSONa, tylko logika wyliczająca kurs i zwracająca dane -- to powinno być normalnie w CurrencyConverterService. Dlaczego zwykły processor JSONów, który dostaje text i ma zwrócić obiekt wie cokolwiek o kalkulatorze cen?